### DISCUSSION TITLE
[FR] The ability to reorder fields in a field group

### DISCUSSION THREAD

**Discussion by mmikkel** [craftcms/cms#8889] [Ideas]
### Description

Related to https://github.com/craftcms/cms/issues/6568:

If the ability to add  field groups is added in the new field layout designer in 3.5 (similar to how this was possible in previous versions) like the #6568 feature request suggest, I'd also suggest making it possible to reorder fields in the field groups, so that if the entire field group is added to a layout, the fields are initially placed in that same order (relative to each other) within the layout.

Being able to easily add a whole field group to the field layout is a nice feature, but it was never really *that* useful, since it's never been possible to set a custom order for fields in a group. More often than not, you had to manually reorder the fields in the field layout after adding the field group, which typically took almost as much time as it would've taken to just manually drag each field into the layout in the first place.

**Comment by bkrsmanovic** [comment]
I agree. And this option to 'Order' field in field group can be set when creating field in group, rather than using Letters or Date of Creation. 
Advanced stuff would be reordering fields in Field Layout, and it remembers new order across sessions, but if we can control order in 1 place, I guess that would be more than enough for now. :)

**Comment by mmikkel** [comment]
To be clear I don't think the order of fields in a particular field group should persist for fields in a field layout, after the fields have been added; i.e. fields already in a layout shouldn't be affected by the order of fields in their field group – that'd be a nightmare to work with (and to implement, I suspect).

I'm only suggesting that it'd be nice to be able to manually order fields within their field groups, so that if/when #6568 is implemented, it'll be easier to create ready-to-use field groups that can be added to various field layouts, without having to always reorder the fields if they are already set up with a sensible default order within their group.

A typical use case for us is a "SEO" field group, with fields for SEO titles, descriptions and images. We add this group to all entry types with a URI, typically by dragging the whole field group into the layout (before 3.5, I mean). But, we don't want the "SEO Description" field above the "SEO Title" field, so we always have to reorder the fields manually after adding the group. It's a small thing, but annoying.
